Makefile

```diff
@@ -44,7 +44,7 @@ test-tier2:
 
 # TIER3
 test-tier3:
-	$(MAKE) test-jira
+	ginkgo -v -focus "Export applications"
```

Comment:
I'd suggest wrap the ginkgo "Export application" call to a Makefile target with name indicating it is a migrationwaves test and call it from the test-tier3. (I'm heading offline for shutdown, so lgtm)